### DISCUSSION TITLE
material you when

### DISCUSSION THREAD

**Discussion by inari-codes** [Sketchware-Pro/Sketchware-Pro#281] [Q&A] [answered]
i know the branch was deleted but all i want is material you, even it's trash but i want to do some experiment with it and doing it manually is pain

**Comment by j-px** [comment, marked as answer]
Material You didn't exactly get deleted, it got merged into the main branch. Which means that the next v6.4.0 test build will allow projects to use it.

**Reply by inari-codes** [reply]
thanks, glad to hear that, also sorry for late reply
